.docker/src-www/etc/nginx/partials/staticsite.conf.tmpl

```diff
@@ -1,4 +1,21 @@
 
+    location / {
+        # Decode the URI for proper matching
+        set $decoded_uri $uri;
+        if ($decoded_uri ~* ^(.+?)(%C2%A0|\xA0|%E2%80%89|\xE2\x80\x89|%E2%80%83|\xE2\x80\x83)+$) {
+            set $clean_uri $1;
+            return 301 $clean_uri;
+        }
+        # Match and strip other partial %C2 sequences from the end of the URI
+        if ($uri ~* ^(.+?)(%C2|\xC2)+$) {
+            set $clean_uri $1;
+            return 301 $clean_uri;
+        }
+
+        # try_files $uri @rewrite; # For Drupal <= 6
```

Comment:
We don't need these try_files lines for the static site. There's no Drupal there and should be no index.php. (Looks like just an accidental copy-n-paste.)

Reply:
oops, you'r right. This is now fixed